scripts/build-one-app-docker-setup.js

```diff
@@ -102,7 +104,7 @@ const doWork = async () => {
     );
   }
 
-  const extraCertsExists = await fs.exists(path.join(sampleProdDir, 'extra-certs.pem'));
+  const extraCertsExists = await fs.access(path.join(sampleProdDir, 'extra-certs.pem'));
```

Comment:
Had to look up why this changed, `fs.exists` is deprecated:
https://nodejs.org/api/fs.html#fsexistspath-callback

Just for future reviewers!